.github/workflows/build-and-test.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
     timeout-minutes: 30
     strategy:
       matrix:
-        go-version: [ 1.14, 1.15 ]
+        go-version: [ 1.17 ]
```

Comment:
You are upgrading the JDK version, are 1.14 and 1.15 still supported?

Reply:
Not support anymore.

Reply:
Then, do we have any compiling doc or something to update?

Reply:
Add the Go version limit in the How-to-compile.md.